pyanaconda/modules/common/structures/requirement.py

```diff
@@ -65,9 +65,15 @@ def name(self, value: Str):
 
     @property
     def reason(self) -> Str:
-        """A reason of the requirement.
+        """An explanation of the requirement.
 
-        Usually a name of a component that requires this requirement.
+        Provides a message that explains why this requirement
+        is required.
+
+        For example:
+            Required to join the realm.
```

Comment:
I would recommend: "Required to join a realm."

Reply:
Fixed.